test/guardian_phoenix/socket_test.exs

```diff
@@ -1,27 +1,30 @@
-defmodule Guardian.Phoenix.SocketTest do
+defmodule Guardian.Phoenix.SocketTests do
```

Comment:
This doesn't need to be plural, the filename isn't.

Reply:
The file and module name do not need to be the same. This helps to read the logs when you run the tests since there is more than one test on the module.

Anyway, if you feel strongly about it, I change it

Reply:
I'd like to change it. I have a PR out to add this rule to Credo, it's in the community styleguide, and it's consistent with the vast majority of code.

Reply:
Interesting, could you share the community style guide?

I wasn't aware that there is any guide for this.

Should I fix `socket_test_modules.ex` as well then?

Reply:
This is the styleguide: https://github.com/christopheradams/elixir_style_guide

Credo PR to introduce this rule: https://github.com/rrrene/credo/pull/587

Reply:
Let's ignore `socket_test_modules` for now. We realistically shouldn't have multiple modules per file either. Save that for a later PR. New code should conform 👌